README.md

```diff
@@ -109,6 +116,20 @@ done instead by editing the `config.yml` we used earlier.
 
 ## Step 4: Run the model
 
+## Bare metal
+
+Assuming you are in environment with a compiled version of the FV3 model at the path `/absolute/path/to/fv3.exe` and a rundirectory at the path `<rundir>`, you can run the model like this
+
+    cd <rundir>
+    mpirun -n <number of processors> /absolute/path/to/fv3.exe
+
+The number of processors has to be `6 * num_tiles` where num_tiles is the product of the `namelist.fv_core_nml.layout` configurations.
+
+
+## Docker
+
+If you would like to run the model in one of the included docker containers use the command
+
```

Comment:
The below name of the docker image is actually wrong, it's probably more something like `us.gcr.io/vcm-ml/fv3gfs-compiled:gnu7-mpich314-nocuda` instead. Maybe just put a note saying that the specific name may differ and is displayed after the `make build` or `make build_debug` command?